Dockerfile.local

```diff
@@ -1,4 +1,4 @@
-FROM node:8
+FROM node:12
```

Comment:
do we need to update `engines`? https://github.com/screwdriver-cd/screwdriver/blob/master/package.json#L26

Reply:
Thx.
I fixed it.